plugins/plugin-java/che-plugin-java-ext-jdt/org-eclipse-jdt-ui/src/main/java/org/eclipse/jdt/internal/ui/text/java/AbstractJavaCompletionProposal.java

```diff
@@ -545,10 +544,9 @@ public String getAdditionalProposalInfo() {
    */
   public Object getAdditionalProposalInfo(IProgressMonitor monitor) {
     if (getProposalInfo() != null) {
-      String info = getProposalInfo().getInfo(monitor);
+      String info = "";
```

Comment:
Did we lose the displaying javadoc in the completion proposals here? If so, we would have to open an issue to restore the functionality.